src/server/ua_server_internal.h

```diff
@@ -65,6 +65,26 @@ typedef enum {
     UA_DIAGNOSTICEVENT_PURGE
 } UA_DiagnosticEvent;
 
+#ifdef UA_ENABLE_FILETYPE_OBJECT_SUPPORT
+typedef struct UA_FileInfo {
+    LIST_ENTRY(UA_FileInfo) listEntry;
+    UA_file *file;
+    UA_UInt32 filehandle;
+    UA_Byte openFileMode;
+}UA_FileInfo;
+
+typedef struct UA_FileType {
+    LIST_ENTRY(UA_FileType) listEntry;
+    UA_NodeId fileTypeObjectNodeId;
```

Comment:
fileTypeObjectNodeId or fileNodeId?

Reply:
Why a list of filetyeobjects in the ua_server_internal.h is necessary?